pkgs/applications/misc/warpd/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libXi
+, libXinerama
+, libXft
+, libXfixes
+, libXtst
+, libX11
+, libXext
+
+, waylandSupport ? false, cairo, libxkbcommon, wayland
+}:
+
+stdenv.mkDerivation rec {
+  pname = "warpd";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "rvaiya";
+    repo = "warpd";
+    rev = "e618c26b721919a97b0fa9da89501fda271ae53d";
+    sha256 = "yfqv+aABPSpw62BXpKXtBOp4CdmZfGSks+PojjsM/Vw=";
+  };
+
+  buildInputs =  [
+    libXi
+    libXinerama
+    libXft
+    libXfixes
+    libXtst
+    libX11
+    libXext
+  ]
+  ++ lib.optionals waylandSupport [ cairo libxkbcommon wayland ]
+  ;
+
+  preConfigure = ''
+    mkdir -p $out/share/man/man1
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m644 warpd.1.gz $out/share/man/man1/
+    install -m755 bin/warpd $out/bin/
+  '';
+
+  meta = {
```

```suggestion
  meta = with lib; {
```
And please remove repeats of `lib`.